app/src/main/res/values/strings.xml

```diff
@@ -926,4 +926,10 @@
     <string name="accept_follow_request_failure">Failed accepting follow request from %1$s: %2$s</string>
     <string name="reject_follow_request_failure">Failed rejecting follow request from %1$s: %2$s</string>
 
+    <string name="reblog_privacy_public_description">Everyone can see your boost</string>
+    <string name="reblog_privacy_unlisted_description">Same as public but it will never be shown in public timelines</string>
+    <string name="reblog_privacy_followers_only_description">Only your followers can see your boost</string>
+
+    <string name="reblog_confirm">Boost this post?</string>
+    <string name="favourite_confirm">Favourite this post?</string>
```

Comment:
Is the base strings file uk english (favourite) or us english (favorite)?

Reply:
It is US, I'll change it to favorite